client/src/get-ticket/components/home.tsx

```diff
@@ -1,14 +1,45 @@
-import { useState } from 'react'
+import { useEffect } from 'react'
```

Comment:
The way you comment the code is useful.
I encourage you to do so always to have much readability in the code.
But, remember a lot of them doesn't work. As a best practice, the code should be written in a way that doesn't need that much comment.